ColorPairUtil.cpp

```diff
@@ -0,0 +1,31 @@
+#include <iostream>
+
+#include "ColorPair.h"
+#include "ColorPairUtil.h"
+
+namespace TelCoColorCoder
+{
+	CColorPair CColorPairUtil::GetColorFromPairNumber(const int &pairNumber)
+	{
+		int zeroBasedPairNumber = pairNumber - 1;
+		MajorColor majorColor = (MajorColor)(zeroBasedPairNumber / MAX_MINOR_COLOR);
+		MinorColor minorColor = (MinorColor)(zeroBasedPairNumber % MAX_MINOR_COLOR);
+		return CColorPair(majorColor, minorColor);
+	}
+
+	int CColorPairUtil::GetPairNumberFromColor(const MajorColor &major, const MinorColor &minor)
+	{
+		return major * MAX_MINOR_COLOR + minor + 1;
+	}
+
+	void CColorPairUtil::PrintAllColorPairs()

```

Comment:
This function isn't called anywhere? How do you know it is working?
Even if you call it, you would need manual verification of 25 lines of output. Can you think of a way to assert its output, to reduce manual testing burden?